sp800-63b/sec4_aal.md

```diff
@@ -82,15 +84,15 @@ The CSP SHOULD employ appropriately tailored security controls from the low base
 
 #### 4.1.5. Records Retention
 
-The CSP shall comply with their respective records retention policies in accordance with applicable laws and regulations. If the CSP opts to retain records in the absence of any legal requirements, the CSP SHALL conduct a privacy risk assessment to determine how long any records should be retained.
+The CSP shall comply with their respective records retention policies in accordance with applicable laws and regulations. If the CSP opts to retain records in the absence of any legal requirements, the CSP SHALL conduct a privacy risk assessment to determine how long records should be retained.
 
 ### 4.2. Authenticator Assurance Level 2
 
-AAL 2 provides high confidence that the claimant controls the authenticator registered to a subscriber. Two different authentication factors are required. Approved cryptographic techniques are required at AAL 2 and above.
+AAL 2 provides high confidence that the claimant controls authenticator(s) registered to the subscriber. Proof of possession and control of two different authentication factors is required through a secure authentication protocol. Approved cryptographic techniques are required at AAL 2 and above.
 
 #### 4.2.1. Permitted Authenticator Types
 
-At AAL 2, it is required to have (a) a multi-factor authenticator, or (b) a combination of two single-factor authenticators. Authenticator requirements are specified in Section 5.
+At AAL 2, it is required to have either a multi-factor authenticator or a combination of two single-factor authenticators. Authenticator requirements are specified in Section 5.
 
 When a multi-factor authenticator is used, any of the following may be used:
```

Comment:
Not a showstopper for Kevin, but we are going to need a hearty reason why 2 SF authenticators can't include a biometric - per our discussion last week with Elaine.